pom.xml

```diff
@@ -454,8 +454,8 @@
                 <configuration>
                     <imageName>${dockerImagePrefix}/${project.artifactId}</imageName>
                     <imageTags>
-                        <imageTag>${project.version}</imageTag>
-                        <imageTag>latest${latestSuffix}</imageTag>
+                        <imageTag>1.15.1-QA-PROXY</imageTag>
```

Comment:
Hmm, this looks like some stub, doesn't?

Reply:
fixed